builders/integrations/oracles/api3.md

```diff
@@ -136,96 +142,47 @@ The data feeds are continuously updated by [first-party oracles](https://docs.ap
 
-### Types of dAPIs {: #types-of-dapis }
-
-There are two types of dAPIs: [self-funded](https://docs.api3.org/reference/dapis/understand/self-funded.html){target=\_blank} and [managed](https://docs.api3.org/reference/dapis/understand/managed.html){target=\_blank}. Managed dAPIs are only available on MainNets, and self-funded dAPIs are available on both MainNets and TestNets. The process to read from a dAPI proxy remains the same for both self-funded and managed dAPIs.
-
-#### Self-funded dAPIs {: #self-funded-dapis }
-
-[Self-funded dAPIs](https://docs.api3.org/reference/dapis/understand/self-funded.html){target=\_blank} are single-source data feeds that are funded by the users with their own funds. They offer developers the opportunity to experience data feeds with minimal up-front commitment, providing a low-risk option prior to using managed dAPIs.
-
-With self-funded dAPIs, you can fund the dAPI with your own funds. The amount of gas you supply will determine how long your dAPI will be available for use. If you run out of gas, you can fund the dAPI again to keep it available for use.
-
-You can read more about [self-funded dAPIs on API3's documentation site](https://docs.api3.org/guides/dapis/subscribing-self-funded-dapis){target=\_blank}.
-
-#### Managed dAPIs {: #managed-dapis }
-
-[Managed dAPIs](https://docs.api3.org/reference/dapis/understand/managed.html) are sourced directly from multiple [first-party](https://docs.api3.org/explore/airnode/why-first-party-oracles.html){target=\_blank} data providers running an Airnode and aggregated using Airnode's signed data using a median function. The gas costs and availability of managed dAPIs are managed by the [API3 DAO](https://docs.api3.org/explore/dao-members){target=\_blank}.
-
-You can read more about [managed dAPIs on API3's documentation site](https://docs.api3.org/reference/dapis/understand/managed.html){target=\_blank}.
+### Subscribing to dAPIs {: #subscribing-to-dapis }
 
-### Access Self-Funded dAPIs {: #access-self-funded-dapis }
+The [API3 Market](https://market.api3.org/moonbeam) lets users access dAPIs on both [Moonbeam Mainnet](https://market.api3.org/moonbeam) and [Testnet](https://market.api3.org/moonbeam-testnet).
 
-The process for accessing self-funded data feeds is as follows:
+#### Exploring, selecting and configuring your dAPI {: #explore-a-dapi }
 
-1. Explore the API3 Market and select a dAPI
-2. Fund a sponsor wallet
-3. Deploy a proxy contract to acess the data feed
-4. Read data from the dAPI
+The [API3 Market](https://market.api3.org/moonbeam) provides a list of all the dAPIs available across multiple chains including testnets. You can filter the list by mainnet or testnet chains. After selecting the chain, you can now search for a specific dAPI by name. Once selected, you will land on the details page (eg ETH/USD on Moonbeam Testnet) where you can find more information about the dAPI.
```

Comment:
Just noticed that the logo for Moonbeam is our old one. Can you please update it to our new logo, which can be found here: https://guidelines.moonbeam.foundation/logo

Is it possible to also change "Moonbeam testnet" to "Moonbase Alpha"? And the amounts for Moonbase are shown in GLMR, can that be updated to DEV? That's what we refer to the testnet token as and it's also what we instruct users to use when configuring their wallets